tools/gen-random-bout/gen-random-bout.cpp

```diff
@@ -22,116 +23,301 @@
 #define stat64 stat
 #endif
 
-// --sym-args 0 1 10 --sym-args 0 2 2 --sym-files 1 8 --sym-stdout
-static int getint(char *i) {
-  if(!i) {
+#define SMALL_BUFFER_SIZE 64 // To hold "arg<N>" string, temporary
+                             // filename, etc.
+
+#define MAX_FILE_SIZES 256
+
+static unsigned get_unsigned(char *i) {
+  if (!i) {
     fprintf(stderr, "ran out of arguments!\n");
     assert(i);
```

Comment:
I'm not sure how this assert could ever fail...

Reply:
The assert always fails.

Reply:
Sorry, my comment was unclear. This should imho be an `exit()` as it only works if asserts are enabled and the assert statement always leads to an abort.

Reply:
@251. This is fixed.